examples/Huggingface_Transformers/Transformer_handler_generalized.py

```diff
@@ -0,0 +1,148 @@
+from abc import ABC
+import json
+import logging
+import os
+import ast
+import torch
+from transformers import AutoModelForSequenceClassification, AutoTokenizer, AutoModelForQuestionAnswering,AutoModelForTokenClassification
+
+from ts.torch_handler.base_handler import BaseHandler
+
+logger = logging.getLogger(__name__)
+
+
+class TransformersSeqClassifierHandler(BaseHandler, ABC):
+    """
+    Transformers handler class for sequence, token classification and question answering.
+    """
+    def __init__(self,ctx):
+        super(TransformersSeqClassifierHandler, self).__init__()
+        self.manifest = ctx.manifest
+        properties = ctx.system_properties
```

Comment:
We could be doing all these inside the initialize method instead of the constructor. `initialize` is called with service context which can be very helpful.